lib/CONST.ts

```diff
@@ -572,6 +585,23 @@ const CONST = {
         VERIFICATION_MAX_ATTEMPTS: 7,
     },
 
+    // The number of days that certain banks limit their transaction imports to
+    // see https://stackoverflow.com/c/expensify/questions/1289/1290#1290 for more about this.
+    // If a bank isn't on this list, we default to 90 days
+    BANK_IMPORT_DAY_LIMITS: {
+        'americanexpress.com': 30,
+        'oauth.bankofamerica.com': 30,
+        'oauth.capitalone.com': 90,
+        'oauth.chase.com': 90,
+        'discover.com': 30,
+        'admin.pexcard.com': 30,
+        svbdirect: 75,
```

Comment:
Should this be a string?

```suggestion
        'svbdirect': 75,
```